### PR TITLE
adds list section component

### PR DESCRIPTION
**Pull request by paaaatrick**
## Trello Card(s)
https://trello.com/c/LYR8wwyc/18-section-list

## What was fixed?
List Section added with styles. 

## Screenshots of fix (visual changes)

Desktop
https://www.dropbox.com/s/wo256qlf5u5qyor/Screenshot%202019-11-08%2010.07.04.png?dl=0

Mobile
https://www.dropbox.com/s/0ic370megfn4774/Screenshot%202019-11-08%2010.27.00.png?dl=0

### CODE REVIEW

File: web/wp-content/themes/fire/acf-layouts/components/list/list.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+  $title = get_sub_field('title');
+  $links = get_sub_field('links');
+
+// echo '<pre>',print_r($links, TRUE),'</pre>';
```
Comment (ryanschmidt) [inline web/wp-content/themes/fire/acf-layouts/components/list/list.php]: Remove commented code

Comment (paaaatrick) [reply web/wp-content/themes/fire/acf-layouts/components/list/list.php]: Will do!

**Comment by BrentWMiller** [comment]
@paaaatrick Let me show you how to export the ACF changes then this will be good.

File: web/wp-config.php [hunk 2]
```diff
@@ -36,7 +36,7 @@
 /**
  * Force SSL
  */
-define( 'FORCE_SSL_ADMIN', true );
+define( 'FORCE_SSL_ADMIN', false );
```
Comment (ryanschmidt) [inline web/wp-config.php]: what is this? @BrentWMiller

Comment (ryanschmidt) [reply web/wp-config.php]: No one cares

Comment (BrentWMiller) [reply web/wp-config.php]: Yeah I need to look into this. It breaks local development. I think I can switch it to a variable

**Pull request merged**